sukhorukov/src/components/Bot.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+export class Bot extends Component{
+  state = {
+    author: "Bot",
+    text: "",
```

Comment:
Определитесь с кавычками - везде разные :)
```suggestion
    text: '',
```